src/main/java/com/rcv/Main.java

```diff
@@ -45,24 +45,26 @@ public static void main(String[] args) {
     try {
       Logger.setup();
     } catch (IOException exception) {
-      System.err.print(String.format("Failed to start system logging: %s", exception.toString()));
+      System.err.print(String.format("Failed to start system logging!\n%s", exception.toString()));
     }
 
     if (args.length == 0) {
       // if no args provided, launch the GUI
       launch(args);
     } else {
       // assume user wants to use CLI
+      Logger.executionLog(Level.INFO, "Tabulator is being used via the CLI.");
       String configPath = args[0];
       // config file for running the tabulator
       ElectionConfig config = loadElectionConfig(configPath);
       if (config != null) {
-        Logger.executionLog(Level.INFO, "Tabulator is being used via the CLI.");
         executeTabulation(config);
       } else {
         Logger.executionLog(Level.SEVERE, "Aborting because config is invalid.");
       }
     }
+
+    System.exit(0);
```

Comment:
Just curious what this does. Won't we exit with a 0 automatically?

Reply:
I was wondering about that too.  Nice fix.